migrations/versions/ec8f94139b02_codify_workspace_name_and_message_id_in_.py

```
@@ -0,0 +1,289 @@
+schema = get_schema()
+
+
+def upgrade() -> None:
+    # Step 1: Drop rows where workspace_name is NULL in payload
+    # This removes invalid/corrupted queue items
+    op.execute(
+        sa.text(
+            f"""
+            DELETE FROM "{schema}".queue
+            WHERE payload->>'workspace_name' IS NULL
+            """
```

Ruff (0.14.2)

Possible SQL injection vector through string-based query construction

(S608)